VCFv4.4.tex

```diff
@@ -625,6 +625,41 @@ \section{INFO keys used for structural variants}
 \end{verbatim}
 \normalsize
 
+\footnotesize
+\begin{verbatim}
+##INFO=<ID=SVCLAIM,Number=A,Type=String,Description="Claim made by the structural variant call. Valid values are CN, BP, CNBP for copy number, breakpoint and both respectively.">
```

Comment:
update to D, J and DJ. also, the verbatim formatting in the pdf makes this to go out of the page. I would just break it in two lines, as the alternative is to put the lines vertically as in other places of the pdf.

Reply:
I support breaking into two lines. IMO having the lines run vertically is terribly unreadable and should be avoided.

Reply:
Will be addressed in #550